test/parallel/test-net-socket-timeout-unref.js

```diff
@@ -11,15 +11,26 @@ server.listen(common.PORT);
 server.unref();
 
 var timedout = false;
+var connections = 0;
+var sockets = [];
 
 [8, 5, 3, 6, 2, 4].forEach(function(T) {
   var socket = net.createConnection(common.PORT, 'localhost');
-  socket.setTimeout(T * 1000, function() {
-    console.log(process._getActiveHandles());
-    timedout = true;
-    socket.destroy();
+  socket.on('connect', function() {
+    if (++ connections === 6) {
```

Comment:
Can you remove the space after `++`. If possible, can you remove the hard coded `6`. Is this just the value of the array length? If so, you can use that.